src/pages/HomePage/HomePage.jsx

```diff
@@ -1,18 +1,47 @@
+import {useEffect, useState} from 'react';
 import AddPost from 'components/UI/AddPost/AddPost';
-import { NormalButton } from 'components/UI/Buttons/buttons';
-import { IconEmojiInput, IconGifInput, IconImageInput, IconsProfile } from 'components/UI/Icons/Icons';
 import React from 'react';
```

Comment:
You can remove this import or add it to line 1.

Reply:
done